### PR TITLE
test(client): add setImmediate to every server socket event to client…

### PR DESCRIPTION
**Pull request by prastoin**
… handler

### CODE REVIEW

**Comment by prastoin** [comment]
PR should now be reviewable

File: packages/client/screens/MusicPlaylistEditorRoomScreen/__tests__/TracksListEditing.test.tsx [hunk 1]
```diff
@@ -4,19 +4,19 @@ import Toast from 'react-native-toast-message';
...
-    render,
+    within,
 } from '../../../tests/tests-utils';
 
 function toTrackCardContainerTestID(id: string): string {
     return `${id}-track-card-container`;
 }
 
-function extractTrackIDFromCardContainerTestID(testID: string): string {
+export function extractTrackIDFromCardContainerTestID(testID: string): string {
```
Comment (Devessier) [inline packages/client/screens/MusicPlaylistEditorRoomScreen/__tests__/TracksListEditing.test.tsx:19]: This is a vicious bug.
When you import something from a test file inside another test file, all the tests defined in the origin file will be duplicated in each file with the import.
If we want to use this function outside of this test file, we should put it in a file that declares no tests.

**Review by Devessier — CHANGES_REQUESTED** [review]
Great refactoring job!
We should change where we export `extractTrackIDFromCardContainerTestID` function to don't duplicate tests in each file we import the function.

File: packages/client/screens/MusicPlaylistEditorRoomScreen/__tests__/TracksListEditing.test.tsx (hunk 1 above)
Comment (prastoin) [reply packages/client/screens/MusicPlaylistEditorRoomScreen/__tests__/TracksListEditing.test.tsx:19]: You're right ! fixing this

**Review by Devessier — APPROVED** [review]
Perfect!
We can merge.

**Pull request merged**